2024 GDG TEAM (#2)
gdsccvr/gdsccvr.github.io: src/pages/team/index.mdx

import { team2021, team2023,team2024} from './_data'
import Team from './_team'

# Team

A team that is extremely innovative and always on toes when it comes to managing and organising an event that can help the student community in their technical endeavours.

## Team 2024-25

> Google Developer Student Clubs CVR 2023-24 team was led by [Sheetal Gujar](https://cshubh.com).

Reviewer: ```suggestion
Google Developer Groups on Campus CVR 2023-24 team is led by [Sheetal Gujar](https://www.linkedin.com/in/sheetal-gujar/).
```

for historical purposes, let's refer to the club by its official name for that term.

<Team data={team2024}/>

## Team 2023-24

> Google Developer Student Clubs CVR 2023-24 team was led by [Shubh A Chudasama](https://cshubh.com).View team members [here](/team/2023)

Reviewer: ```suggestion
Google Developer Student Clubs CVR 2023-24 team was led by [Shubh A Chudasama](https://cshubh.com). View team members [here](/team/2023)
```
